composeApp/src/androidMain/kotlin/com/theolm/instaclone/components/BottomBarComponent.kt

```diff
@@ -0,0 +1,124 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Favorite
+import androidx.compose.material.icons.outlined.Add
+import androidx.compose.material.icons.outlined.FavoriteBorder
+import androidx.compose.material.icons.outlined.Home
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.theolm.instaclone.R
+
+@Composable
+fun BottomBarComponent() {
+    val homeTabState = remember {
+        mutableStateOf(true)
+    }
+    val searchTabState = remember {
+        mutableStateOf(false)
+    }
+    val likeTabState = remember {
+        mutableStateOf(false)
+    }
+    val profileTabState = remember {
+        mutableStateOf(false)
+    }
+    var profileBorderColor by remember {
+        mutableStateOf(Color.Transparent)
+    }
+    val tabList = listOf(homeTabState, searchTabState, likeTabState, profileTabState)
+
+    val iconModifier = Modifier
+        .width(22.dp)
+        .height(24.dp)
+    Row(
+        modifier = Modifier.background(Color.White).fillMaxWidth().height(45.dp).border(
+            BorderStroke(width = 0.5.dp, color = Color(0x1A000000))
+        ),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceAround
+    ) {
+        IconButton(onClick = {
+            selectSingleTab(homeTabState, tabList)
+        }) {
+            Icon(
+                painterResource(if (homeTabState.value) R.drawable.home_selected else R.drawable.home_unselected),
+                modifier = iconModifier,
+                contentDescription = "Home Icon"
+            )
+        }
+        IconButton(onClick = {
+            selectSingleTab(searchTabState, tabList)
+        }) {
+            Icon(
+                painterResource(if (searchTabState.value) R.drawable.search_selected else R.drawable.search_unselected),
```

Comment:
**#question**
Did you try to run this on iOS?
I am asking this because it will probably crash. It is because we are not supporting iOS as well. It is fine for now, after we add the Moko Resources library, we will be able to support both platform 🤗

Reply:
That 100% skipped my mind 🤡 . I have no idea how to. Maybe it's one topic to go ver the next meeting or in private